web/src/app/browser/src/contextManager.ts

```diff
@@ -85,11 +79,88 @@ export default class ContextManager extends ContextManagerBase<BrowserConfigurat
       });
     });
 
-    // TBD:  keyman.domManager.init (the page-integration parts)
-    // CTRL+F: `// Exit initialization here if we're using an embedded code path.`
-    // EVERYTHING after that block will likely go here - DOMManager's role always
-    // was context-management and the facilitation thereof.
-    throw new Error('Method not implemented.');
+    this.engineConfig.deferForInitialization.then(() => {
+      const device = this.engineConfig.hostDevice;
+      const noPropagation = (event: Event) => event.stopPropagation()
+
+      // For any elements being attached, or being enabled after having been disabled...
+      this.page.on('enabled', (elem) => {
+        if(!(elem._kmwAttachment.interface instanceof DesignIFrame)) {
+          // For anything attached but (design-mode) iframes...
+
+          // This block:  has to do with maintaining focus.
+          if(device.touchable) {
+            // Remove any handlers for "NonKMWTouch" elements, since we're enabling it here.
+            this.domEventTracker.detachDOMEvent(elem, 'touchstart', this.nonKMWTouchHandler);
+
+            // Prevent base-page touch handlers from causing a defocus when interacting
+            // with attached input elements.
+            this.domEventTracker.attachDOMEvent(elem, 'touchmove', noPropagation, false);
+            this.domEventTracker.attachDOMEvent(elem, 'touchend', noPropagation, false);
+          }
+
+          // This block:  has to do with maintaining focus.
+          this.domEventTracker.attachDOMEvent(elem,'focus', this._ControlFocus);
+          this.domEventTracker.attachDOMEvent(elem,'blur', this._ControlBlur);
+          this.domEventTracker.attachDOMEvent(elem,'click', this._Click);
+        } else {
+          // For design-mode iframes:
+
+          // This block:  has to do with maintaining focus.
+          var Lelem=(elem as HTMLIFrameElement).contentWindow.document;
+          // I2404 - Attach to IFRAMEs child objects, only editable IFRAMEs here
+          if(device.browser == 'firefox') {
+            this.domEventTracker.attachDOMEvent(Lelem,'focus', this._ControlFocus);
+            this.domEventTracker.attachDOMEvent(Lelem,'blur', this._ControlBlur);
+          } else { // Chrome, Safari
+            this.domEventTracker.attachDOMEvent(Lelem.body,'focus', this._ControlFocus);
+            this.domEventTracker.attachDOMEvent(Lelem.body,'blur', this._ControlBlur);
+          }
```

Comment:
Idea / "feature request" for future improvement:

While working on this, I felt like it'd be wise to add a few properties or methods to the element-wrapper types in some fashion.  This may motivate merging `engine/attachment` and `engine/element-wrappers`, as this would be putting details for the former onto the objects provided by the latter.

Rough ideas:
- `getFocusElement()`:  returns the element expected to handle focus-events when the element is attached.
- The implementation of this would allow the two branches of the if-else ladder to be merged.
    - The `DesignIFrame` type would be responsible for providing `Lelem` and/or `Lelem.body` for attachment-event setup... nice and centralized with all other code specific to supporting design-mode iframes.

If we wanted to go further...
- #8803 added the modularized `outputTargetForElement` type.
- If each element-specific class had code for how to tell if an element is attached, or if it is an appropriate subelement of an attached element, it would probably go a long way to handling more extreme cases.
    - For example, for design-mode iframes, we can detect attachment from a text node that's a direct child... but if there's much more hierarchy than that, we're at a loss.
    - It's probably a far-future point, but something like this may make a potential solution for integrating with something like Google Docs more viable, as it'd keep the relevant code well-encapsulated.

Definitely not something to do for now, but I thought it may be worth noting and documenting, likely as an issue.  Obviously, _this_ isn't an issue - just thought I'd wait to see reviewers' thoughts on this before I actually add it as one, rather than add something that might be instantly discarded.